### DISCUSSION TITLE
not sure if this can be integrated

### DISCUSSION THREAD

**Discussion by gsgoldma** [Mikubill/sd-webui-controlnet#128] [Ideas]
but it looks like it can fine tune a lot of stuff a bit better than control net, at least in different ways, if im not mistaken

https://github.com/TencentARC/T2I-Adapter

**Comment by Mikubill** [comment]
Seems like the structure is similar but adapted on unet's encode phrase. 
<img width="1057" alt="algo" src="https://user-images.githubusercontent.com/31246794/219544925-dc6af0e3-5f8b-4b08-91e4-7f1af6913e00.png">
Will try if we could integrate it.

**Comment by matrix4767** [comment]
Apparently this supports dual model guidance, that is 2 models at once, which would be superb.
But this has the Apache license, so is it even doable?

**Reply by gsgoldma** [reply]
cant believe you already integrated some of it! way to go!
